app/models/admin_ability.rb

```diff
@@ -142,6 +142,10 @@ def signed_in_with_organizer_role(user, conf_ids_for_organization_admin = [])
     can :manage, Room, venue: { conference_id: conf_ids }
     can :manage, Sponsor, conference_id: conf_ids
     can :manage, SponsorshipLevel, conference_id: conf_ids
+    can :manage, Survey, surveyable_type: 'Conference',
+                         surveyable_id:   conf_ids
+    can :manage, SurveyQuestion, survey: { surveyable_type: 'Conference',
+                                           surveyable_id:   conf_ids }
```

Comment:
This has now crossed the threshold of `Metrics/AbcSize` but I’m not sure how to address it.

Reply:
https://github.com/openSUSE/osem/pull/2546#issuecomment-513542629 has the same question

Reply:
`Metrics/AbcSize` is so far just an undiscussed number in `.rubocop_todo.yml`. Just increase it...

Reply:
I see, thanks.